collectors/ebpf.plugin/README.md

```diff
@@ -203,6 +203,37 @@ Linux metrics:
     -   Number of calls to `shm_dt`. (`shmdt_call`)
     -   Number of calls to `shm_ctl`. (`shmctl_call`)
 
+#### Allocate memory
+
+The `allocate memory` specifies how collectors will manage the memory during runtime to monitor application and cgroup. 
+The following options are accepted:
+
+-  `static`:  This option is preferred for high loaded or dedicated servers. It will use more memory, but it needs less
+   iteration with kernels. This is the default option.
+-  `dynamic`:  When the plugin runs with this option, it will allocate memory for a specific PID when the PID is used, but
+   it needs to use a minimum necessary for these future PIDs. Because it will allocate during runtime, it will
+   need more iteration with the kernel. This is the preferred method when the host has a small amount of memory.
+
+The following table demonstrates a comparison between memory usage for each method on a 64 bit computer running a Linux 
+distribution that allows the maximum of `32768` PIDs (Value obtained from `/proc/sys/kernel/pid_max`). 
+
+|  Thread   | Size (bytes) | Static | Dynamic |
+|-----------|--------------|--------|---------|
+| cachestat |           32 | 262144 |  1310720|
+| dcstat    |           64 | 262144 |  2359296|
+| fd        |           32 | 262144 |  1310720|
+| process   |           80 | 262144 |  2883584|
+| shm       |           32 | 262144 |  1310720|
+| socket    |           80 | 262144 |  2883584|
+| swap      |           16 | 262144 |   786432|
+| vfs       |          112 | 262144 |  3932160|
+
+Dynamic allocates the minimum necessary by default, this value is calculated multiplying the total number of PIDs (`32768`)
+times 8 bytes (64 bits).
+
+Static will allocate everything necessary to process when the collector starts. The value for this column is
+calculated adding the value of `Static` to the result of multiplying the column `Size (bytes)` and the number of PIDs (`32768`).
```

Comment:
I'm not sure this part is needed at all.

Reply:
I really doubt the PR fixes the bug, discussed in #12719. @vobruba-martin, could you test the PR? If it doesn't fix the bug, I'm not sure it brings much value. @vkalintiris, what do you think?

Reply:
I sent this information for documentation team and we decided to give the maximum information possible for users to understand how we calculated the values.

Reply:
@DShreve2 what do you think about @vlvkobal suggestion?